src/_includes/install/ulimit.md

```diff
@@ -1,12 +1,12 @@
-### Why to set ulimit
+### Why to set open files ulimit
```

Comment:
The original heading is bad English grammar
```suggestion
### Why open files ulimit should be set
```

Reply:
Hi @BarnyShergold , thank you for reviewing :)

I have implemented your suggestion, could you please approve this PR?
Thank you!